src/config/locales/gd/translation.json

```diff
@@ -0,0 +1,542 @@
+    "reblog_error": "Cha deach leis a’ bhrosnachadh",
+    "unreblog_error": "Cha b’ urrainn dhuinn am brosnachadh a thoirt air falbh",
+    "favourite_error": "Cha b’ urrainn dhuinn a chur ris na h-annsachdan",
+    "unfavourite_error": "Cha b’ urrainn dhuinn a thoirt air falbh o na h-annsachdan",
+    "bookmark_error": "Chaidh a chur ris na comharran-lìn",
+    "unbookmark_error": "Cha deach leinn an comharra-lìn a thoirt air falbh",
+    "delete_error": "Cha b’ urrainn dhuinn am post a sguabadh às",
+    "search_error": "Dh’fhàillig leis an lorg",
+    "toot_error": "Dh’fhàillig leis a’ phostadh",
+    "update_list_memberships_error": "Cha b’ urrainn dhuinn na ballrachdan liosta ùrachadh",
+    "add_user_error": "Cha b’ urrainn dhuinn an cleachdaiche a chur ris",
+    "authorize_url_error": "Cha b’ urrainn dhuinn an t-URL ùghdarrachaidh fhaighinn",
+    "domain_confirmed": "Chaidh {{domain}} a dhearbhadh, clàraich a-steach a-nis",
+    "domain_doesnt_exist": "Cha deach leinn ceangal a dhèanamh le {{domain}}, dearbh URL an fhrithealaiche",
+    "loading": "’Ga luchdadh…",
+    "language_not_support_spellchecker_error": "Cha chuir an dearbhair-litreachaidh taic ris a’ chànan seo",
+    "update_filter_error": "Cha b’ urrainn dhuinn a’ chriathrag ùrachadh",
+    "create_filter_error": "Dh’fhàillig le cruthachadh na criathraige"
+  },
+  "validation": {
+    "login": {
+      "require_domain_name": "Tha feum air ainm àrainne",
+      "domain_format": "Na cuir a-steach ach ainm na h-àrainne"
+    },
+    "new_toot": {
+      "toot_length": "Bu chòir dhan phost a bhith eadar {{min}} is {{max}} a dh’fhaid",
+      "attach_length": "Chan urrainn dhut barrachd air {{max}} dealbh(an) a cheangal ris",
+      "attach_image": "Chan urrainn dhut ach dealbhan no videothan a cheangal ris",
+      "poll_invalid": "Roghainnean mì-dhligheach sa chunntas-bheachd"
+    }
+  },
+  "notification": {
+    "favourite": {
+      "title": "Annsachd",
+      "body": "Is annsa le {{username}} am post agad"
+    },
+    "follow": {
+      "title": "Leantainn ort",
+      "body": "Tha {{username}} a’ leantainn ort a-nis"
+    },
+    "follow_request": {
+      "title": "Iarrtas leantainn",
+      "body": "Fhuair thu iarrtas leantainn o {{username}}"
+    },
+    "reblog": {
+      "title": "Brosnachadh",
+      "body": "Bhrosnaich {{username}} am post agad"
+    },
+    "quote": {
+      "title": "Iomradh",
+      "body": "Thug {{username}} iomradh air a’ phost agad"
+    },
+    "reaction": {
+      "title": "Freagairt",
+      "body": "Fhreagair {{username}} dhan phost agad"
+    },
+    "status": {
+      "title": "Post",
+      "body": "Tha {{username}} air rud a phostadh"
+    },
+    "poll_vote": {
+      "title": "Bhòt cunntais-bheachd",
+      "body": "Cuir {{username}} bhòt ris a’ chunntas-bheachd agad"
+    },
+    "poll_expired": {
+      "title": "Crìoch cunntais-bheachd",
+      "body": "Thàinig an cunntas-bheachd aig {{username}} gu crìoch"
+    }
+  }
+}
```

Comment:
Don't edit `translation.json` file. This file will be automatically updated by crowdin. So please write words on crowdin.

Reply:
OK, will do. Don't worry about merging this and having it overwritten, I can reconstruct everything in my translation software :)

Reply:
Thank you very much and I apologize for the extra work. Crowdin let me upload the json file, so the translation is now complete over there :)

I have opened 2 source string issues at Crowdin.